src/components/modal/ModalCreateBooks.css

```diff
@@ -266,7 +274,8 @@
   z-index: 2;
   width: 450px;
   height: 900px;
-  box-shadow: 0 1px 12px rgb(0 0 0 / 50%);
+  box-shadow: 0 1px 12px rgba(0, 0, 0, 0.5);
+  display: block;
   background-color: wheat;
```

Comment:
you can use different colours, you don't have to use built in ones :)

Reply:
Ok ok